specs/agents/tracing-instrumentation-db.md

```diff
@@ -188,6 +188,21 @@ If they don't, they MUST use a hard-coded list of URLs that correspond with the
 |`_.type`| `mongodb` | |
 |`_.name`| e.g. `customers` | Database name, same as `db.instance` if available  |
 
+#### `mongodb_capture_statement_commands` configuration
+
+Agents that support capturing MongoDB statements MUST implement this option.
+
+|                |  |
+|----------------|--|
+| Type           | `List<`[`WildcardMatcher`](../../tests/agents/json-specs/wildcard_matcher_tests.json)`>` |
+| Default        | `"find,aggregate,count,distinct,mapReduce"` |
+| Central config | `false` |
+
+Examples:
+- capture all statements: `*`
```

Comment:
```suggestion
- capture for all commands: `*` (This should be discouraged, because it can lead to capturing sensitive information in `insert` and `update` commands.)
```